bundles/org.openhab.binding.iaqualink/src/main/java/org/openhab/binding/iaqualink/internal/config/IAqualinkConfiguration.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.iaqualink.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Configuration properties for connecting to a iAqualink Account
+ *
+ * @author Dan Cunningham - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class IAqualinkConfiguration {
+
+    /**
+     * user to us when connecting to the account
+     */
+    public @NonNullByDefault({}) String userName;
```

Comment:
I'm not sure it was meant for fields like this

cc: @maggu2810

Reply:
Hm, as this members are set by reflection (`asClass`) I assume they can be null and so @Nullable

Reply:
Yes, i hastily added annotation here without thinking about.